vendor.conf

```diff
@@ -33,7 +33,7 @@ github.com/tchap/go-patricia v2.2.6
 gopkg.in/cheggaaa/pb.v1 v1.0.7
 gopkg.in/inf.v0 v0.9.0
 gopkg.in/yaml.v2 v2
-github.com/docker/docker da99009bbb1165d1ac5688b5c81d2f589d418341
+github.com/docker/docker 86f080cff0914e9694068ed78d503701667c4c00
```

Comment:
This seems to be rolling Docker back from [2018-05-22][1] to [2017-10-18][2].  Was the rollback intentional?  Can we avoid bumping Docker for this PR?

[1]: https://github.com/moby/moby/commit/da99009bbb1165d1ac5688b5c81d2f589d418341
[2]: https://github.com/moby/moby/commit/86f080cff0914e9694068ed78d503701667c4c00

Reply:
Fixed ... not sure what the deal was there.